packages/config/src/bridges/sygma.ts

```diff
@@ -0,0 +1,282 @@
+import { EthereumAddress, ProjectId, UnixTime } from '@l2beat/shared-pure'
+
+import { NUGGETS } from '../common'
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { RISK_VIEW } from './common'
+import { Bridge } from './types'
+
+const discovery = new ProjectDiscovery('sygma')
+
+export const sygma: Bridge = {
+  type: 'bridge',
+  id: ProjectId('sygma'),
+  display: {
+    name: 'Sygma',
+    slug: 'sygma',
+    category: 'Hybrid',
+    description:
+      'Sygma is a modular, open-source, cross-consensus interoperability protocol enabling asset transfers, non-fungible tokens, and decentralized finance. With Sygma, developers can easily extend their applications across Ethereum mainnet, Base, Cronos, Polygon, Gnosis, Polkadot, Kusama, and other Substrate-based chains with active work on Bitcoin and Cosmos SDK interoperability.',
```

Comment:
You are talking about two different things here: Types of data (Tokens) and use cases. I suggest to focus on one: ...enabling transfers of fungible and non-fungible assets, general message passing (GMP/AMB) as well as cross-chain execution'

Reply:
nabbed from the whitepaper, but i can modify per your suggestion. updated.